HEN_HOUSE/egs++/egs_run_control.cpp

```diff
@@ -810,10 +997,27 @@ EGS_RunControl *EGS_RunControl::getRunControlObject(EGS_Application *a) {
     }
     else {
         if (a->getNparallel() > 0) {
-            result = new EGS_JCFControl(a);
+           vector<string> allowed_types;
+           allowed_types.push_back("simple");
+           allowed_types.push_back("uniform");
+           allowed_types.push_back("balanced");
+           int rco_t = irc->getInput("rco type",allowed_types,2);
```

Comment:
Can I change the "rco type" input key to simply "type"; I prefer that the input keys don't have cryptic abbreviations in them. Here "type" seems sufficient in the `run control` input block, consistent with "type" in other inputs.

Reply:
Sure!

Reply:
@ftessier could we merge this one?